### PR TITLE
Added Note to Contributors

### PR DESCRIPTION
**Pull request by BryanBailey11**
please look over this note and let me know of any additions or changes you think need to be done. Thanks! I added the same note to both the Readme and the Code of Conduct.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
 
 # Contributor Covenant Code of Conduct
 
+# Note to Contributors
+
+Due to certain uncontrollable circumstances surrounding the environment in which
+Architecture as code is being developed and funded, we cannot accept any new outside
+contributors at this time. We will reject any pull requests from untrusted sources.
```
Comment (cvchaparro) [inline CODE_OF_CONDUCT.md:8]:
```suggestion
contributions at this time. We will reject any pull requests from unknown sources.
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 [![Main branch AaC Workflow](https://github.com/jondavid-black/AaC/actions/workflows/main-branch.yml/badge.svg)](https://github.com/jondavid-black/AaC/actions/workflows/main-branch.yml)
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/from-referrer/)
+# Note to Contributors
```
Comment (cvchaparro) [inline README.md:3]:
```suggestion

# Note to Contributors
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+Due to certain uncontrollable circumstances surrounding the environment in which
+Architecture as code is being developed and funded, we cannot accept any new outside
+contributors at this time. We will reject any pull requests from untrusted sources.
```
Comment (cvchaparro) [inline README.md:7]:
```suggestion
contributions at this time. We will reject any pull requests from unknown sources.
```

**Review by cvchaparro — CHANGES_REQUESTED** [review]
Looks good, overall, just a few minor recommendations.

**Review by cvchaparro — APPROVED** [review]
Nice work!

**Review by Coffee2Bits — APPROVED** [review]
Looks good. 👍

**Pull request merged**